SDKCLI/Sources/SDKCLI/Utils/Process.swift

```diff
@@ -0,0 +1,59 @@
+import Foundation
+
+extension Process {
+    convenience init(_ argsList: [String]) {
+        self.init()
+        self.executableURL = URL(fileURLWithPath: "/usr/bin/env")
+        self.arguments = argsList
+    }
+    
+    convenience init(_ args: String...) {
+        self.init(Array(args))
+    }
+}
+
+extension Process {
+    struct Git {
```

Comment:
thought we could use https://github.com/SwiftGit2/SwiftGit2

sadly No SPM support.